modules/efs/test/main.tf

```diff
@@ -0,0 +1,14 @@
+module "efs" {
```

Comment:
please have tests within tests folder and have named folders for each test,
for example this test setup can be placed in ./tests/general-purpose-efs/ folder,

please use relative path for source "dasmeta/modules/aws//modules/efs" =>"../.."